maps/biter_battles_v2/feeding_calculations.lua

```diff
@@ -179,6 +179,9 @@ function Public.calc_send_command(
                 if v == 'aquilo' or starts_with(v, 'cr') then
                     v = 'cryogenic-science-pack'
                 end
+                if v == 'promethium' or v == 'dark' or starts_with(v, 'pr') then
```

Comment:
promethium starts with pr, so can delete first expression
```suggestion
                if starts_with(v, 'pr') or v == 'dark' then
```